src/types/onyx/Policy.ts

```diff
@@ -32,6 +32,10 @@ type Policy = {
     isFromFullPolicy?: boolean;
     lastModified?: string;
     customUnits?: Record<string, unknown>;
+
+    areChatRoomsEnabled?: boolean;
+
+    isPolicyExpenseChatEnabled?: boolean;
```

Comment:
NAB, are all Onyx types supposed to be optional or something? I'm kinda confused about that part. Seems like maybe certain fields would always be there (e.g. the `id`) yet it is still optional 🤔

Reply:
Yeah I agree, to be completely honest `areChatRoomsEnabled` and `isPolicyChatEnabled` shouldn't be optional, from my understanding of how ReconnectApp works, but since I saw that the ID was optional I decided to just mark them as optional 🤷

Reply:
@hayata-suenaga do you have any thoughts on this?

Reply:
sorry for the late reply. Properties that you can guarantee to always exist should'n be optional.

Please fix the optional type declarations when you spot them 🙇

Reply:
fixed!